vllm/distributed/communication_op.py

```diff
@@ -315,3 +322,35 @@ def broadcast_tensor_dict(
         for async_handle in async_handles:
             async_handle.wait()
     return tensor_dict
+
+
+def send_next_rank(tensors: List[torch.Tensor]) -> None:
+    """Send the tensors to the next pipeline model parallel rank."""
+    pynccl_comm = get_pp_pynccl_communicator()
```

Comment:
when do we use pynccl vs regular torch? can you comment here?

Reply:
We don't end up using this anymore!

Reply:
PyNCCL was used with CUDAGraph in previous versions of this PR though.